Q/qwt/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "qwt"
+version = v"6.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ig-or/qwt.git", "30812fc4a2630ec9956f4daa6c4f6d4b331c74a6")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd qwt
+qmake
+make -j 
+make install
+cp $prefix/lib/qwt.dll $prefix/bin/qwt.dll
+cp $prefix/lib/qwt.dll $prefix/bin/libqwt.dll
+cp $prefix/lib/qwtd.dll $prefix/bin/qwtd.dll
+cp $prefix/lib/qwtd.dll $prefix/bin/libqwtd.dll
+cd $WORKSPACE/srcdir
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Windows(:x86_64),
+    Linux(:x86_64, libc=:glibc)
```

Comment:
Sorry, here we're using a slightly different syntax for the platforms
```suggestion
    Platform("x86_64", "windows"),
    Platform("x86_64", "linux"; libc="glibc"),
```